port/common/omrstr.c

```diff
@@ -2303,14 +1915,48 @@ omrstr_free_tokens(struct OMRPortLibrary *portLibrary, struct J9StringTokens *to
  * If buf is too small, will return the minimum buf size required.
  */
 uint32_t
-omrstr_ftime(struct OMRPortLibrary *portLibrary, char *buf, uint32_t bufLen, const char *format, int64_t timeMillisUTC)
+omrstr_ftime_ex(struct OMRPortLibrary *portLibrary, char *buf, uint32_t bufLen, const char *format, int64_t timeMillisUTC, uint32_t flags)
 {
-	if (buf && bufLen > 0) {
-		return omrstr_subst_time(portLibrary, buf, bufLen, format, timeMillisUTC);
+	if ((NULL != buf) && (bufLen > 0)) {
+		return omrstr_subst_time(portLibrary, buf, bufLen, format, timeMillisUTC, flags);
 	}
 	return 0;
 }
 
+/**
+ * Returns the UTC time that was passed in as a formatted string in local time.  Formatted according to the 'format' parameter.
+ *
+ * @param[in] portLibrary  The port library.
+ * @param[in,out] buf A pointer to a character buffer where the resulting time string will be stored.
+ * @param[in] bufLen The length of the 'buf' character buffer.
+ * @param[in] format The format string, ordinary characters placed in the format string are copied.
+ * to buf without conversion.  Conversion specifiers are introduced by a '%' character, and are replaced in buf as follows:.
```

Comment:
Same problem with "to buf without conversion."

Reply:
The problem is that I added the 'flags' documentation in the middle of the description for 'format'. I'll move it where it belongs.